build/configs/rtl8730e/loadable_apps_xip/defconfig

```diff
@@ -27,6 +28,7 @@ CONFIG_NUM_APPS=2
 CONFIG_SUPPORT_COMMON_BINARY=y
 CONFIG_COMMON_BINARY_VERSION="200204"
 CONFIG_COMMON_BINARY_NAME="common"
+CONFIG_COMMON_BIN_DYN_RAMSIZE=524288
```

Comment:
What does it mean? A common binary uses separate heap?

Reply:
Its the space in RAM we allocate for data and bss sections of common binary.

Reply:
I am currently using the same configs CONFIG_*_BIN_DYN_RAMSIZE for the space allocated for each app or common in the ram. If its common that space is used only for data and bss. If its app, its used for data, bss and heap. I think using the name DYN_RAMSIZE for common is wrong for common. I will change it to CONFIG_COMMON_BIN_STATIC_RAMSIZE

Reply:
Yes, we need to rename it as you said.
But do we allocate a data and bss in `CONFIG_APP1_BIN_DYN_RAMSIZE`??

Reply:
Yes, data and bss for apps are allocated in CONFIG_APP1_BIN_DYN_RAMSIZE itself. We first allocate data and bss, then rest of it is allocated as heap.

Reply:
As discussed, we will take up changing the desing to remove fixed CONFIG_COMMON_BIN_STATIC_RAMSIZE